src/v2/BaseLoginRouter.js

```diff
@@ -253,6 +272,18 @@ export default Router.extend({
     this.controller.render();
   },
 
+  unsetAttributes () {
+    this.settings.unset('stateToken');
+    this.settings.unset('useInteractionCodeFlow');
+    this.settings.unset('proxyIdxResponse');
+  },
+
+  updateStateHandleInStorage (stateToken) {
+    if(!sessionStorage.getItem('okta-siw-state-token') && stateToken) {
```

Comment:
Create utility which has
- constant for string `okta-siw-state-token`
- read/write/delete state token from session storage

Or
- constant for string `okta-siw-state-token` into `src/v2/view-builder/utils/Constants.js` and move it to upper folder like `src/v2`

Reply:
changed, but i will move it to `src/v2` in a different PR.. It's been used at various places.